index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../css/styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <img src="../src/assets/images/header-shapes-mobile@2x.svg" alt="background" class="bg_shape">
+        <nav>
+            <ul class="flex-nav">
+                <li class="logo">
+                    <a href="#"><img alt="logo" src="../src/assets/images/my_logo.png"></a>
+                </li>
+                <li class="menu">
+                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="wrap">
+        <h1>I am TEEJAY! Happy to have you here today!!</h1>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+                <a href="#">LET'S CONNECT</a>
```

Comment:
- [OPTIONAL]  kindly use the correct font as the one provided on the Figma design. You can import this `Poppins` font from [google fonts](https://fonts.google.com/specimen/Poppins?query=poppins). :wink: